mailbox/memory/src/main/java/org/apache/james/mailbox/inmemory/InMemoryMessageId.java

```diff
@@ -77,8 +76,6 @@ public final boolean equals(Object obj) {
 
     @Override
     public String toString() {
-        return MoreObjects.toStringHelper(InMemoryMessageId.class)
-                .add("value", value)
-                .toString();
+        return serialize();
```

Comment:
Why? I guess the previous code would help us to catch some bad .toString() usage.